.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug Report
+about: Create a bug report
+title: Bug Report
+labels: bug
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+<!--- Look through existing open and closed issues to see if someone has reported the issue before -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Screenshots (if appropriate)
+
+## Context
+<!--- How has this issue affected you? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Amunsen version used:
+* Browser Name and version:
```

Comment:
Normally the issue is on the backend or ingestion or external connection. I felt the browser section as well as OS may not be very relevant.

Reply:
Makes sense! Removing those.

Reply:
hey @Golodhros , the Browser and Operating System are still there?